tinyagentos/routes/device_pair_requests.py

```
@@ -0,0 +1,213 @@
+        if notifs is not None:
+            try:
+                await notifs.add(
+                    title="Pair request",
+                    message=f"{display} wants to connect (code {verify_code})",
+                    level="warning",
+                    source="pair_requests",
+                    user_id=admin_id,
+                    data={"request_id": pair_request_id, "platform": body.platform},
+                )
+            except Exception:
+                pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)